platform/fabric/services/state/recipients.go

```diff
@@ -99,13 +99,16 @@ func (f RequestRecipientIdentityView) Call(context view.Context) (interface{}, e
 		return nil, err
 	}
 
+	timeout := time.NewTimer(time.Minute)
+	defer timeout.Stop()
```

Comment:
sometimes defer is used, other times no. Why this difference?

Reply:
you cannot use defer in a loop